src/h5web/Breadcrumbs.tsx

```diff
@@ -0,0 +1,68 @@
+import { Fragment, ReactElement, useContext } from 'react';
+import { FiChevronRight } from 'react-icons/fi';
+import styles from './BreadcrumbsBar.module.css';
+import { assertAbsolutePath } from './guards';
+import { ProviderContext } from './providers/context';
+
+interface Props {
+  path: string;
+  onSelect: (path: string) => void;
+  showDomain: boolean;
+}
+
+function Breadcrumbs(props: Props): ReactElement {
+  const { path, onSelect, showDomain } = props;
+
+  assertAbsolutePath(path);
+  const { domain } = useContext(ProviderContext);
+
+  if (path === '/') {
```

Comment:
Having `Breadcrumbs` in its own component allows for an early return if the root is selected. In this case, we don't need to generate crumbs or to check if the domain should be shown.